.gitignore

```diff
@@ -1,5 +1,40 @@
-.DS_Store
-run_server.sh
-.sass-cache/*
-_site/
-.idea/
+.DS_STORE
+node_modules
+scripts/flow/*/.flowconfig
+*~
+*.pyc
+.grunt
+_SpecRunner.html
+__benchmarks__
+build/
+remote-repo/
+coverage/
+.module-cache
+fixtures/dom/public/react-dom.js
+fixtures/dom/public/react.js
+test/the-files-to-test.generated.js
+*.log*
+chrome-user-data
+*.sublime-project
+*.sublime-workspace
+.idea
+*.iml
+.vscode
+*.swp
+*.swo
+
+babel.config.js
+jest.config.js
+yarn.lock
+.eslintrc.js
```

Comment:
```
babel.config.js
jest.config.js
.eslintrc.js
```

These configuration files are required, the build fails without them. Remove these lines from `.gitignore` then commit these three files.